app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -30,15 +31,26 @@ class TaxonTreeVis extends React.Component {
 
     this.metrics = {
       aggregatescore: {
-        label: "Aggregate Score",
         agg: arr => Math.max(...arr)
       },
-      nt_r: { label: "NT r", agg: arr => arr.reduce((a, b) => a + b, 0) },
-      nt_rpm: { label: "NT rpm", agg: arr => arr.reduce((a, b) => a + b, 0) },
-      nt_zscore: { label: "NT Z-Score", agg: arr => Math.max(...arr) },
-      nr_r: { label: "NR r", agg: arr => arr.reduce((a, b) => a + b, 0) },
-      nr_rpm: { label: "NR rpm", agg: arr => arr.reduce((a, b) => a + b, 0) },
-      nr_zscore: { label: "NR Z-Score", agg: arr => Math.max(...arr) }
+      nt_r: {
```

Comment:
Sorry for another ask, but we do no need the objects now. Can be just a mapping of metrics to aggregators.

Reply:
What do you mean by objects?

Reply:
oh i see